addons/web/models/models.py

```diff
@@ -189,7 +189,10 @@ def cleanup(vals: Dict) -> Dict:
                         co_record = co_record.with_context(**field_spec['context'])
 
                     if 'fields' in field_spec:
-                        reference_read = co_record.web_read(field_spec['fields'])
+                        try:
+                            reference_read = co_record.web_read(field_spec['fields'])
+                        except AccessError:
+                            reference_read = [{'id': co_record.id, 'display_name': '_NoAccess_'}]
```

Comment:
I don't know if this was discussed but maybe we'd want this value to be translated?